### PR TITLE
Profile patch

### PR DESCRIPTION
**Pull request by LinguList**
We have been using code to generate language-specific profiles for some time now. The code that we used to make language-specific profiles form "normal" profiles was always copy-pasted, and it is time to add it to pylexibank.

The code is EXTREMELY easy: we load the profile as a dictionary, we iterate over Graphemes in the data, and we add each IPA segment we find there. 

The typical workflow now is:

1. make a general profile for a dataset
2. use `cldfbench lexibank.language_profiles DATASET` to make language-specific profiles
3. edit those profiles

### CODE REVIEW

**Comment by LinguList** [comment]
I have still not really learned how to do tests in these commandline apps. So excuse me for not adding, but if one of you could add exemplary tests later, I would add them next time.

**Review by LinguList — COMMENTED** [review]
Note that the additional code in check_profile is also important, as it provides access to "slashed" items.

**Review by SimonGreenhill — APPROVED** [review]
looks good to me!

File: src/pylexibank/commands/language_profiles.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        for form in language.forms:
+            for grapheme in form.data["Graphemes"].split():
+                data[grapheme, profile.get(grapheme, "?")] += 1
+        new_path = ds.etc_dir / "orthography" / "{0}.tsv".format(language.id)
+        if new_path.exists() and not args.force:
+            raise ParserError("Orthography profile exists, use --force to override")
+        with open(
+                new_path,
+                "w",
+                encoding="utf-8") as f:
```
Comment (johenglisch) [inline src/pylexibank/commands/language_profiles.py:50]: This throws an error in fresh datasets where the folder `etc/orthography` doesn't exist yet.

File: src/pylexibank/commands/language_profiles.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        for row in reader:
+            profile[row["Grapheme"]] = row["IPA"]
+
+    
+    for language in progressbar(
+            wordlist.objects("LanguageTable"), 
+            desc="creating profiles"):
+        data = defaultdict(int)
+        for form in language.forms:
+            for grapheme in form.data["Graphemes"].split():
```
Comment (johenglisch) [inline src/pylexibank/commands/language_profiles.py:42]: This throws an error in datasets where the `Graphemes` column is empty (e.g. [hillburmish](https://github.com/lexibank/hillburmish)):

      File "[...]/pylexibank/src/pylexibank/commands/language_profiles.py", line 42, in run
        for grapheme in form.data["Graphemes"].split():
    AttributeError: 'NoneType' object has no attribute 'split'

**Comment by LinguList** [comment]
Thanks, these have been addressed now, I throw an error for hillburmish, as we cannot make a profile for individual languages there.

**Review by johenglisch — APPROVED** [review]

**Pull request merged**